converters/README.md

```diff
@@ -10,13 +10,13 @@ Please sign up at https://slack.lfai.foundation/ and join [onnx-converters](http
 
 # SIG Leads
 
-* Guenther Schmuelling (Microsoft)
+* Thiago Crepaldi (Microsoft)
 * Kevin Chen (NVIDIA)
 
 # Logistics
 
 * SIG leads will drive the meeting.
-* Meeting annoucement will be posted in our gitter channel: https://gitter.im/onnx/converters
+* Meeting annoucement will be posted in our Slack channel
```

Comment:
nit: perhaps provide the link to onnx-converters channel? (https://lfaifoundation.slack.com/archives/C0171FSKZBN)

Reply:
I didn't include it here because it's linked up above in the Slack channel section.